lib/prototype-rails/on_load_action_view.rb

```diff
@@ -14,9 +14,11 @@
   include ActionView::Helpers::ScriptaculousHelper
 end
 
-ActionView::TestCase.class_eval do
-  include ActionView::Helpers::PrototypeHelper
-  include ActionView::Helpers::ScriptaculousHelper
+if defined?(Rails) && Rails.env.test?
```

Comment:
The problem with this is if users is testing in a cucumber environment it will not load these helpers.